src/modelconfig.cpp

```diff
@@ -42,6 +42,22 @@ bool ModelConfig::isReloadRequired(const ModelConfig& rhs) const {
         SPDLOG_DEBUG("ModelConfig {} reload required due to name mismatch", this->name);
         return true;
     }
+    if (this->stateful != rhs.stateful) {
+        SPDLOG_DEBUG("ModelConfig {} reload required due to stateful type mismatch", this->name);
```

Comment:
Replace SPDLOG_DEBUG with SPDLOG_LOGGER_DEBUG(modelmanager_logger.

Similar case for following lines.